package.json

```diff
@@ -31,7 +31,8 @@
     "eject": "react-scripts eject",
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "postinstall": "patch-package"
+    "postinstall": "patch-package",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\""
```

Comment:
please avoid changing the packages.... (can cause merge conflict for others PR's)